src/test/java/ibm/jceplus/junit/base/BaseTestHmacMD5.java

```diff
@@ -277,22 +246,4 @@ public void testHmacMD5_key7() throws Exception {
         assertTrue("Mac digest did not equal expected", Arrays.equals(digest, digest_7));
 
     }
-
-    //--------------------------------------------------------------------------
-    //
-    //
-    public void warmup() throws Exception {
```

Comment:
Don't we need this `warmup()` method?

Reply:
I dont anticipate using this test as a performance test. I dont believe we need to loop in this test to perform a "warmup".